CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### DEV
```

Comment:
We typically add a changelog entry when making a new release. For example, https://github.com/nginxinc/nginx-prometheus-exporter/commit/f0173677183c840e90a56e48082e36ac687e1a30 There is no need to modify this file.